app/classifier/index.cjsx

```diff
@@ -655,6 +672,8 @@ module.exports = React.createClass
       <Classifier {...@props}
         workflow={@props.workflow}
         subject={@state.subject}
+        guide={@state.guide}
```

Comment:
Ok ignore my comment. I confused myself because the index file is the classifier and the classifier wrapper components. We need to split these out.